Implementations/Libuavcan/Platforms/index.md

```diff
@@ -4,10 +4,71 @@ weight: 20
 
 # Libuavcan platforms
 
-These sections are dedicated to officially supported platforms and also provide documentation
+This section is dedicated to the officially supported platforms and also provide documentation
 on how to add support for a new platform.
 
-Sources for all official platform drivers are located in the directory
-[`libuavcan_drivers`](https://github.com/UAVCAN/libuavcan/tree/master/libuavcan_drivers).
+Official platform drivers are located in dedicated repositories here:
+[github.com/UAVCAN](https://github.com/UAVCAN).
 
 Contributions adding support for new platforms are always welcome.
+
+## Adding support for a new platform
+
+{% include lightbox.html url="/Implementations/Libuavcan/figures/libuavcan_data_flow.png" title="Libuavcan data flow" thumbnail=true %}
+
+Implementing a platform driver for libuavcan is generally a quite straightforward task,
+especially if the driver is targeted for a certain application and need not to be generic.
+
+Essentially, a libuavcan driver is a set of C++ classes implementing the following C++ interfaces:
+
+* `uavcan::ICanDriver`
+* `uavcan::ICanIface`
+* `uavcan::ISystemClock`
+
+Listed interfaces are defined in
+[`libuavcan/include/uavcan/driver/`](https://github.com/UAVCAN/libuavcan/blob/master/libuavcan/include/uavcan/driver).
+
+Note that the library core already includes a non-blocking prioritized TX queue, so normally the driver should not
```

Comment:
"so normally the driver should not implement software TX queue itself." -> "so normally A driver should not
+implement A software TX queue itself."